docs/en/sql-reference/aggregate-functions/reference/argmax.md

```diff
@@ -49,3 +50,60 @@ Result:
 │ director             │
 └──────────────────────┘
 ```
+
+**Extended example**
+
+```sql
+CREATE TABLE test
+(
+    a Nullable(String),
+    b Nullable(Int64)
+)
+ENGINE = Memory AS
+SELECT *
+FROM values(('a', 1), ('b', 2), ('c', 2), (NULL, 3), (NULL, NULL), ('d', NULL));
+
+select * from test;
+┌─a────┬────b─┐
+│ a    │    1 │
+│ b    │    2 │
+│ c    │    2 │
+│ ᴺᵁᴸᴸ │    3 │
+│ ᴺᵁᴸᴸ │ ᴺᵁᴸᴸ │
+│ d    │ ᴺᵁᴸᴸ │
+└──────┴──────┘
+
+select argMax(a, b), max(b) from test;
+┌─argMax(a, b)─┬─max(b)─┐
+│ b            │      3 │ -- argMax = b because it the first not-Null value, max(b) is from another row!
+└──────────────┴────────┘
+
+select argMax(tuple(a), b) from test;
+┌─argMax(tuple(a), b)─┐
+│ (NULL)              │ -- Tuple allows to get Null value.
```

Comment:
```suggestion
│ (NULL)              │ -- The a `Tuple` that contains only a `NULL` value is not `NULL`, so the aggregate functions won't skip that row because of that `NULL` value
```